File: semester2/2/programming/lab4/task2.cpp
#include <iostream>

using namespace std;

float calculateWallsAndFloorArea(float a,float b,float h) {return ((b*h+a*h)*2 + a*b);}

int main() {
    float a,b,h;
    cout << "a = ";
    cin >> a;
    cout << "b = ";
    cin >> b;
    cout << "h = ";
    cin >> h;
    cout << calculateWallsAndFloorArea(a,b,h) << endl;
    return 0;
}

File: semester2/2/programming/lab4/task2.pas
Program lab4_2;
Var a,b,h:real;
function calculateWallsAndFloorArea(a,b,h:real):real;
begin
  calculateWallsAndFloorArea := ((b*h+a*h)*2 + a*b);
end;
Begin
  write('a = ');
  read(a);
  write('b = ');
  read(b);
  write('h = ');
  read(h);
  writeln(calculateWallsAndFloorArea(a,b,h));
End.